rdbms/src/main/java/io/spine/server/storage/jdbc/query/IdColumn.java

```diff
@@ -285,7 +287,7 @@ public String normalize(String id) {
 
         @Override
         public Type sqlType() {
-            return column().type() == STRING ? STRING : STRING_255;
+            return column().type() == STRING ? STRING : STRING_512;
```

Comment:
I don't understand why we can't say `return column.type()` here. It seems always to be one of the following: `STRING`, `STRING_255`, `STRING_512`. 

And is it OK that this method can return `STRING_512` when the actual type is `STRING_255`?